### ISSUE TITLE
scrollDefault: 'now'

### ISSUE THREAD

**Issue by pbirkhead** [jonthornton/jquery-timepicker#457]
The scrollDefault option isn't working for me.  The first entry is always 12:00 AM.  I even echoed my system's time to make sure it was actually returning a time, which it is.

    $(document).ready(function(){
        $("#start_time, #due_time").timepicker({ 
            scrollDefault: 'now', 
            interval: '15'
        });
    });

I know the function is working because the interval option is working.  I can also successfully do this:

    $(document).ready(function(){
        $("#start_time, #due_time").timepicker({ 
            startTime: new Date(0,0,0,8,0,0), // 8:00:00 AM
            interval: '15'
        });
    });

I saw in an example where scrollDefault is placed in single quotes, and I tried that without success as well.  Is there an issue with the plugin or with my code?  I have verified the issue in Safari (9.0.1), Chrome (46.0.2490.80) and Firefox (42.0).

Here is the HTML that the jQuery functions are referencing:
```HTML
<div class="formRow">
    <div class="inputLabel">
        <label class="labelText" for="start_date">Start date:</label>
    </div>
    <div style="float:left">
        <input id="start_date" class="date_data" type="text" name="start_date"/><span><input id="start_time" class="time_data" type="text" name="start_time"/></span>
    </div>
    <div class="clear"></div>
</div>

<div class="formRow">
    <div class="inputLabel">
        <label class="labelText" for="due_date">Due date:</label>
    </div>
    <div style="float:left">
        <input id="due_date" class="date_data" type="text" name="due_date"/><input id="due_time" class="time_data" type="text" name="due_time"/></span>
    </div>
    <div class="clear"></div>
</div>
```

**Comment by jonthornton** [comment]
Can you post this as a [jsFiddle](http://jsfiddle.net/jonthornton/28uvg/)?

**Comment by pbirkhead** [comment]
Hi Jon,

Thanks for taking a look at this.

Here’s my jsFiddle attempt.  http://jsfiddle.net/pbirkhead/ps1t2ejk/1/#&togetherjs=4blbvbhQ19

I’ve never posted anything on jsFiddle before.  I tried to post this one but there are a some things going on I’m not sure how to handle.  In the HEAD of my document I reference a couple of CDNs.  Important to this particular page are Google’s jQuery library and accompanying style sheet, and your plugin and accompanying style sheet which I have in a directory on my local machine.  When I tried to include the HEAD in the HTML section of the jsFiddle, it told me not to.  And then I have the issue of your plugin code which resides on my machine, not on a CDN.  I don’t know how to resolve that obstacle with jsFiddle either.

As a result, your timepicker plugin doesn’t work at all in the jsFiddle.  (The DatePicker plugin also has no styling, so obviously I’m not setting up the jsFiddle right.)  I know the timepicker plugin is working in my code, because I can get it to do everything I want but the scrollDefault: ‘now’ option.  Other options work fine.  The environment I am using right now is MAMP.  I can move it up to my hosted environment if that would help you tell me what is wrong.  Just let me know.

Here’s the HEAD of my document in case you want to review it.  Not all of it is necessary for the particular page I’m trying to use timepicker plugin on.

<!DOCTYPE html>
<html>
<head>
    <title>eniture CRM</title>
    <meta charset="UTF-8">
    
    
    <link rel="stylesheet" href="css/style.css">

    
    <script src="http://ajax.googleapis.com/ajax/libs/jquery/1.11.1/jquery.min.js"></script>
    <link rel="stylesheet" href="//ajax.googleapis.com/ajax/libs/jqueryui/1.11.1/themes/smoothness/jquery-ui.css" />

    
    <script src="//ajax.googleapis.com/ajax/libs/jqueryui/1.11.1/jquery-ui.min.js"></script>
    
    
    <script src="https://cdn.datatables.net/1.10.9/js/jquery.dataTables.min.js"></script>
    <link rel="stylesheet" href="//cdn.datatables.net/1.10.9/css/jquery.dataTables.min.css"/>
    
    
    <script src="//cdnjs.cloudflare.com/ajax/libs/moment.js/2.8.4/moment.min.js"></script>
    <script src="//cdn.datatables.net/plug-ins/1.10.10/sorting/datetime-moment.js"></script>
    

    
    <script src="functions/jfunc.js"></script>
    
    
    <script src="functions/jquery.timepicker.js"></script>
    <link rel="stylesheet" href="css/jquery.timepicker.css">

</head>


Sincerely,
Paul Birkhead
-----------------------------------
T: 404.787.6037
E: paul.birkhead@icloud.com

**Comment by jonthornton** [comment]
Try forking my example jsFiddle instead. Everything's already set up for you: http://jsfiddle.net/jonthornton/28uvg/

The trick is to use the "External Resources" section to load scripts and stylesheets. Hosted versions of the timepicker JS and CSS can be found on CDNJS: https://cdnjs.com/libraries/jquery-timepicker

If it's easier to make your code publicly accessible, that also works.

**Comment by pbirkhead** [comment]
Hi Jon,

I found the problem.  The library I downloaded is version 1.3.2, not 1.8.7.   When I switched from the file to the CDN, everything worked fine.

I’m not exactly sure how I got the old version.  I downloaded it just a couple of days ago.  I double checked the ZIP file on GitHub and it contained version 1.8.7.  Very odd.

Thanks for your help!


Sincerely
Paul Birkhead
------------------------------------
Eniture Technology
T: 404.787.6037
F: 770.692.0278
paul.birkhead@eniture.com <mailto:paul.birkhead@eniture.com>
